packages/prisma/migrations/20250502113828_setup_triggers_for_routing_form_response_denormalized/migration.sql

```diff
@@ -0,0 +1,306 @@
+-- Add index for bookingAssignmentReason (lowercase)
+CREATE INDEX "RoutingFormResponseDenormalized_bookingAssignmentReason_idx" ON "RoutingFormResponseDenormalized" (LOWER("bookingAssignmentReason"));
```

Comment:
Is there any reason these indexs live outside of the first migration? Can we move them to the main migration and out of the trigger logic?

Reply:
I split the migrations into three files:

1. Creating tables and indexes
2. Creating triggers to fill in RoutingFormResponseDenormalized
3. Creating triggers to fill in RoutingFormResponseField

Everything combined in a single file might be too overwhelming and hard to see what's doing what. But do you think we should merge them for some reason?